seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/split/FileSourceSplitEnumerator.java

```diff
@@ -56,45 +55,54 @@ public void open() {
 
     @Override
     public void run() {
-        pendingSplit = getHiveFileSplit();
-        assignSplit(context.registeredReaders());
+        // do nothing
```

Comment:
line#52  `assignedSplit` cannot be re -initialized, it may contain restore state